templates/advantage/index.html

```diff
@@ -0,0 +1,240 @@
+{% extends "advantage/base_advantage.html" %}
+
+{% block title %}Ubuntu Advantage for Infrastructure{% endblock %}
+{% block meta_description %}Ubuntu Advantage for Infrastructure offers a single, per-node packaging of the most comprehensive software, security and IaaS support in the industry, with OpenStack support, Kubernetes support included, and Livepatch, Landscape and Extended Security Maintenance to address security and compliance concerns.{% endblock %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1AW8wQu1Uvq0xFtd_c2JwNdic075MPOTx7k2KHqFPPeo/edit{% endblock meta_copydoc %}
+
+{% block content %}
+<section class="p-strip--suru-half-and-half is-dark">
+  <div class="row">
+    <div class="col-8">
+      <h1>Ubuntu Advantage for Infrastructure</h1>
+      <p class="p-heading--four">Ubuntu Advantage for Infrastructure is the most comprehensive Linux enterprise subscription, covering all aspects of open infrastructure.</p>
+    </div>
+  </div>
+</section>
+
+
+{% if openid %}
+<section class="p-strip--light">
+  <div class="row">
+    <div class="col-10">
+      <h2>You are signed in as {{ openid.fullname }} ({{ openid.email }})</h2>
+    </div>
+    <div class="col-2">
+        <a href="/logout" class="u-float-right u-no-margin--right p-button--neutral">Sign out</a>
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="col-8 u-sv3">
+      <h3>Your personal subscription</h3>
+      <p>You are entitled to use Ubuntu Advantage Essential on 3 machines.</p>
+      <aside class="p-accordion" role="tablist" aria-multiselect="true" style="border: 1px solid #cdcdcd;">
+        <ul class="p-accordion__list" style="margin-bottom: 0;">
+          <li class="p-accordion__group">
+            <button type="button" class="p-accordion__tab" id="tab1" role="tab" aria-controls="#tab1-section" aria-expanded="false">Show instructions</button>
+            <section class="p-accordion__panel" id="tab1-section" role="tabpanel" aria-hidden="true" aria-labelledby="tab1-section" style="padding: 0 1rem;">
+              <p>To attach a machine:</p>
+              <div class="p-code-copyable">
+                <input class="p-code-copyable__input" value="sudo ua attach {{ personal_account.free_token }}" readonly="readonly">
+                <button class="p-code-copyable__action">Copy to clipboard</button>
+              </div>
+
+              <p>To detach:</p>
+              <div class="p-code-copyable">
+                <input class="p-code-copyable__input" value="sudo ua detach {{ personal_account.free_token }}" readonly="readonly">
+                <button class="p-code-copyable__action">Copy to clipboard</button>
+              </div>
+            </section>
+          </li>
+        </ul>
+      </aside>
+    </div>
+  </div>
+
+  {{ accounts }}
```

Comment:
This is dumping everything into the view template:

<img width="1267" alt="Screenshot 2019-09-06 at 08 17 13" src="https://user-images.githubusercontent.com/505570/64409063-12202a80-d080-11e9-98e0-c3d5f1e47e2a.png">

Reply:
Removed